sdk/src/Services/SQS/Custom/Model/GetQueueAttributesResponse.extensions.cs

```diff
@@ -178,6 +172,28 @@ public string Policy
             }
         }
 
+        /// <summary>
+        /// Whether or not the queue is a First-In-First-Out (FIFO) queue
+        /// </summary>
+        public bool? FifoQueue
```

Comment:
Initially I suggested having the new properties be `bool?`, but giving it more thought I think simply `bool` is more appropriate for `FifoQueue`, since that property can only have two values: either the queue is FIFO or not.